### PR TITLE
Swap container to be stream8 based

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,23 @@
-FROM centos:7
+FROM centos:stream8
```
Comment (evgeni) [inline Dockerfile:1]:
```suggestion
FROM quay.io/centos/centos:stream8
```

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
 
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
-    yum -y install epel-release git && \
-    yum -y install python2-pip && \
+    yum -y install git glibc-langpack-en epel-release && \
```
Comment (evgeni) [inline Dockerfile:4]: we shouldn't need `epel-release` anymore (pip is in centos these days)

Comment (ehelms) [reply Dockerfile:4]: Truth. Koji is in epel though.

Comment (evgeni) [reply Dockerfile:4]: Huh, yeah.

File: Dockerfile [hunk 3]
```diff
@@ -1,20 +1,22 @@
-FROM centos:7
+FROM quay.io/centos/centos:stream8
 
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
-    yum -y install epel-release git && \
-    yum -y install python2-pip && \
+    yum -y install git glibc-langpack-en epel-release && \
+    yum -y install python3-pip && \
```
Comment (evgeni) [inline Dockerfile:5]: You could fold this into the previous line, saving one yum transaction. But not tooo important.

**Review by evgeni — APPROVED** [review]

**Pull request merged**